upgrade/network/network_upgrade_1.2_to_1.3.md

```diff
@@ -0,0 +1,125 @@
+# Management Network Upgrade CSM 1.2 to 1.3
+
+- [Management Network Upgrade CSM 1.2 to 1.3](#management-network-upgrade-csm-12-to-13)
+  - [Management Network Changelog](#management-network-changelog)
+  - [When to perform switch upgrades](#when-to-perform-switch-upgrades)
+  - [Prerequisites](#prerequisites)
+  - [Upgrade CANU to the latest version](#upgrade-canu-to-the-latest-version)
+  - [Backup running switch configurations](#backup-running-switch-configurations)
+  - [Retrieve SLS data](#retrieve-sls-data)
+  - [Generate CSM 1.3 switch configurations](#generate-csm-13-switch-configurations)
+  - [Discover CSM 1.2 to CSM 1.3 configuration changes](#discover-csm-12-to-csm-13-configuration-changes)
+  - [Test the network prior to update](#test-the-network-prior-to-update)
+  - [Apply and save CSM 1.3 switch configurations](#apply-and-save-csm-13-switch-configurations)
+  - [Test the network after update](#test-the-network-after-update)
+
+## Management Network Changelog
+
+CSM 1.3 includes full support for the Bifurcated CAN feature, specifically deployment of user traffic on the high speed network (CHN). Other switch related changes for the release are minimal.
+
+- Removal of CMN VLAN from UAN to complete BICAN cut-over
```

Comment:
Is it CMN or CAN vlan that is being removed from UAN?

Reply:
This is actually the CMN in this case.  In CSM 1.2 upgrade we created CAN and moved traffic to CAN but didn't lock out the CMN from UAN until CSM 1.3.  Reads weird but true.  I'll add some more text.